src/color-picker/README.md

```diff
@@ -42,6 +42,30 @@ As with other components, you can listen to the `colorchange` event:
 </color-picker>
 ```
 
+### Dynamic
+
+All attributes are reactive:
+
+```html
+<label>
+	Space:
+	<select id="space_select" size="3"></select>
+</label>
+
+<color-picker id="dynamic_picker" space="oklch" color="oklch(60% 30% 180)"></color-picker>
+
+<script type="module">
+	import Color from "https://colorjs.io/dist/color.js";
+
+	space_select.innerHTML = Object.entries(Color.spaces)
+		.map(([id, space]) => `<option value="${id}">${space.name}</option>`)
+		.join('\n');
+
+	space_select.oninput = () => dynamic_picker.space = space_select.value;
+
+	customElements.whenDefined("color-picker").then(() => space_select.value = dynamic_picker.space.id);
```

Comment:
Actually why do we need this at all? We know what the space is, it's oklch, because we defined it ourselves!

I think it's good to try and keep demos to very very few lines of code.
You can also add `Color` as a static property on `ColorPicker` and then we can get rid of the first line too (though then we have to await whenDefined again)